src/stan/services/util/create_rng.hpp

```diff
@@ -22,7 +22,7 @@ namespace stan {
         // Advance generator to avoid process conflicts
         static boost::uintmax_t DISCARD_STRIDE
           = static_cast<boost::uintmax_t>(1) << 50;
-        rng.discard(DISCARD_STRIDE * (chain - 1));
+        rng.discard(DISCARD_STRIDE * (chain));
```

Comment:
The -1 was there before to make sure chain 1 (lowest legal value in the interfaces) didn't advance the PRNG.  Now we're going to use the PRNG to generate transformed data and then advance chain 1 by 2^50 draws.